libraries/AP_Proximity/AP_Proximity_LD06.h

```diff
@@ -0,0 +1,104 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * ArduPilot device driver for Inno-Maker LD06 LiDAR
+ *
+ * ALL INFORMATION REGARDING PROTOCOL WAS DERIVED FROM LD06 DATASHEET:
+ *
+ * http://wiki.inno-maker.com/display/HOMEPAGE/LD06?preview=/6949506/6949511/LDROBOT_LD06_Development%20manual_v1.0_en.pdf
+ *
+ * Author: Adithya Patil, Georgia Institute of Technology
+ * Based on the SLAMTEC RPLiDAR code written by Steven Josefs, IAV GmbH
+ *
+ */
+
+
+#pragma once
+
+#include "AP_Proximity.h"
```

Comment:
```suggestion
#include "AP_Proximity_config.h"
```

... and include the actual header below